### ISSUE TITLE
Remote storage

### ISSUE THREAD

**Issue by sgtnasty** [nnasa14/recipes#11]
Use a remote database to store and retrieve recipes (like mysql or postgresql).

**Comment by nnasa14** [comment]
Would sqlite3 work to accomplish this?

**Comment by sgtnasty** [comment]
Yes, but it requrires lots of work arounds for remote access. That is why you use an [ORM](https://en.wikipedia.org/wiki/Object%E2%80%93relational_mapping), it lets you switch database engines with minimal fuss.

**Comment by nnasa14** [comment]
Added remote_storage.py which contains sqlalchemy instance  042e0c2b775b1dd3edc02abc0e43dab53f05a570
